app/models/classes/format_url.rb

```diff
@@ -0,0 +1,18 @@
+class Classes::FormatUrl
```

Comment:
My 2 pennies before Dan replies: my guess would be just a file called post_helpers where you place the format url methods. Then they remain used as they were in the index file. Just require it in rails helper (I think). Then if it is no longer a class, your last test should be moved to post_spec as a unit test, as it will test the behavior of a post (with special abilities, but still a post).